### PR TITLE
(frozen) v0.3: default DEGREE=20

### PR DESCRIPTION
**Pull request by lispc**
changes:

1. set default super circuit degree to 20, fixing "rw not enough" panic. Proving time increased to ~30min.
2. use uncompressed point in params.  Loading time of params26 reduce from 10minutes to 10seconds in a 8 core server.  But file size increased from 4G to 8G.
3. decouple version of params and circuit(vk, verifier.sol) since they actually are not same thing.
4. re-enable zktrie in super circuit.   Need to deploy https://github.com/scroll-tech/go-ethereum/pull/247 first!
5. use latest version of public input hashing mechanism

release files:

```
 $ aws  --profile default s3 ls s3://circuit-release/circuit-release/params-0320/
2023-03-31 01:23:08  134217988 params20
2023-03-31 01:23:08 8589934852 params26
2023-03-31 01:23:08        202 sha256sum
[  1:27AM ]  [ lispc@dev1:~/into_s3/circuit-release ]
 $ aws  --profile default s3 ls s3://circuit-release/circuit-release/release-v0.3/
2023-03-31 01:21:55       8263 full_proof.data
2023-03-31 01:21:55        328 sha256sum
2023-03-31 01:21:55      36995 verifier.sol
2023-03-31 01:21:55        744 verify_circuit.vkey
```

### CODE REVIEW

**Comment by lispc** [comment]
oh substrate upgrade their ecdsa dependency... the Cargo.lock is broken now...

**Comment by lispc** [comment]
TODO:

after Jerry update all traces, merge this branch into main, and tag "v0.3" on main

**Comment by kunxian-xia** [comment]
We should update the `contrib/download_params.sh` script as it's out-dated.

File: zkevm/Cargo.toml
Comment (HAOYUatHZ) [inline zkevm/Cargo.toml:1]: we can update the `version` in `Cargo.toml`s to `0.3.0`

**Pull request merged**